### PR TITLE
Yd hm19

### PR DESCRIPTION
**Pull request by ydanko**

### CODE REVIEW

File: src/test/java/Homework19.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+import org.testng.Assert;
+
+import javax.swing.*;
+
+
+public class Homework19 extends BaseTest {
+
+    @Test
+    public void DeletePlaylist() throws InterruptedException {
+        login("dankoyanka@gmail.com", "te$t$tudent");
```
Comment (Apte-m) [inline src/test/java/Homework19.java:13]: you can use dataProvaider

File: src/test/java/Homework19.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+        WebElement Playlist = driver.findElement(By.xpath
+                ("//li[@class='playlist playlist']"));
```
Comment (Apte-m) [inline src/test/java/Homework19.java:16]: you can use css selector .playlist.playlist

File: src/test/java/Homework19.java [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+                (By.xpath("//button[@class='del btn-delete-playlist']"));
+        delbtn.click();
+        Thread.sleep(2000);
+
+        WebElement ConfirmBtn = driver.findElement
+                (By.xpath("//button[@class='ok']"));
+        ConfirmBtn.click();
+        Thread.sleep(2000);
+
+        WebElement Popup = driver.findElement(By.xpath
```
Comment (Apte-m) [inline src/test/java/Homework19.java:30]: popUp lowerCase for variable

File: src/test/java/Homework19.java [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+        WebElement delbtn = driver.findElement
+                (By.xpath("//button[@class='del btn-delete-playlist']"));
+        delbtn.click();
+        Thread.sleep(2000);
+
+        WebElement ConfirmBtn = driver.findElement
+                (By.xpath("//button[@class='ok']"));
+        ConfirmBtn.click();
+        Thread.sleep(2000);
```
Comment (Apte-m) [inline src/test/java/Homework19.java:28]: you can use wait wait.until(ExpectedConditions.invisibilityOfElementLocated(popUp);

Comment (ydanko) [reply src/test/java/Homework19.java:28]: This command will be learned in the next lesson

Comment (ydanko) [reply src/test/java/Homework19.java:28]: do I need to apply the changes and re-submit?

Comment (Apte-m) [reply src/test/java/Homework19.java:28]: no need to resubmit

**Pull request closed without merging**